shared/chat/conversation/info-panel/index.js

```diff
@@ -72,13 +72,13 @@ type AddPeopleRow = {
   teamname: string,
 }
 
-type ParticipantRow = {
+type ParticipantRow = {|
```

Comment:
flow warns you if you spread an object it needs its type to be exact or it has no idea what fields will be in the output